README.md

```diff
@@ -117,13 +120,15 @@ git commit -m 'docs: add details about commitlint in README.md'
 Find out more at [conventional-changelog/commitlint](https://github.com/conventional-changelog/commitlint#what-is-commitlint).
 
 ## Contact
-- Open [Github Issues](https://github.com/gosling-lang/gosling.js/issues/) to ask questions or request features.
+
+-   Open [Github Issues](https://github.com/gosling-lang/gosling.js/issues/) to ask questions or request features.
 
 ## Team
-- Sehi L'Yi (<sehi_lyi@hms.harvard.edu>)
-- Qianwen Wang (<qianwen_wang@hms.harvard.edu>)
-- Fritz Lekschas ([@flekschas](https://twitter.com/flekschas) | [lekschas.de](https://lekschas.de))
-- Nils Gehlenborg (<nils@hms.harvard.edu>)
+
+-   Sehi L'Yi (<sehi_lyi@hms.harvard.edu>)
```

Comment:
Just curious -- Are these empty spaces and new lines added in purpose?

Reply:
Sorry for the confusion. I have installed some MD formatters in my vscode and they automatically format the file when I edit/save the file.